CRM/Mailing/BAO/MailingJob.php

```diff
@@ -520,7 +520,8 @@ public function deliver(&$mailer, $testParams = NULL) {
                         AND     $edTable.id IS null
                         AND     $ebTable.id IS null
                         AND    contact_a.is_opt_out = 0 
-                        $aclWhere";
+                        $aclWhere
+                    ORDER BY    $contactTable.contact_type";
```

Comment:
A solution like 
`
ORDER BY IF($contact.contact_type = 'Individual', 0, 1)
`
would fit the needs more exactly. But it increases time consumption by factor 10.